packages/terra-card/themes/clinical-lowlight-theme/clinical-lowlight-theme.scss

```diff
@@ -0,0 +1,16 @@
+:global {
+  // ---------------------- Shared ----------------------- //
+  --terra-card-background-color: #222a2e;
+  --terra-card-border: 1px solid #181b1d;
+  --terra-card-box-shadow: none;
+  --terra-card-color: #b2b5b6;
+  --terra-card-raised-box-shadow: 0 3px 8px -2px rgba(0, 0, 0, 0.18), 0 3px 10px 0 rgba(0, 0, 0, 0.12), 0 3px 3px -3px rgba(0, 0, 0, 0.3);
+
+  // -------------------- Version 3.x -------------------- //
```

Comment:
Can this be removed? It thought our themes were no longer tied to version anymore

Reply:
Addressed here : https://github.com/cerner/terra-core/pull/2758/commits/28a7c8dc409e0e443329574f2c3b8e220c7fa250